packages/mui-codemod/src/v5.0.0/base-hook-imports.test/actual.js

```diff
@@ -0,0 +1,8 @@
+import { useAutocomplete } from '@mui/base/AutocompleteUnstyled';
+import { useBadge } from '@mui/base/BadgeUnstyled';
+import { useButton } from '@mui/base/ButtonUnstyled';
+import { useInput } from '@mui/base/InputUnstyled';
+import { useSelect, SelectOption, SelectChild, isOptionGroup, SelectOptionGroup } from '@mui/base/SelectUnstyled';
```

Comment:
```suggestion
import SelectUnstyled, { SelectUnstyledProps, useSelect, SelectOption, SelectChild, isOptionGroup, SelectOptionGroup } from '@mui/base/SelectUnstyled';
```

Let's ensure that the imports related to SelectUnstyled will not be lost.